codetransformer/code.py

```diff
@@ -260,14 +260,14 @@ def from_pycode(cls, co):
                 ) for b in Bytecode(co)
             ),
         )
-        for idx, instr in enumerate(instrs):
+        for idx, instr in enumerate(sparse_instrs):
             if instr is None:
                 # The sparse value
                 continue
             if instr.absjmp:
-                instr.arg = instrs[instr.arg]
+                instr.arg = sparse_instrs[instr.arg]
             elif instr.reljmp:
-                instr.arg = instrs[instr.arg + idx + 3]
+                instr.arg = sparse_instrs[instr.arg + idx + 3]
```

Comment:
Relative jumps don't count the current opcode plus the two bytes for the argument.